packages/app/package.json

```diff
@@ -53,14 +55,18 @@
     "function-runner": "4.0.2",
     "graphql-request": "5.2.0",
     "http-proxy": "1.18.1",
-    "javy-cli": "0.1.4",
+    "javy-cli": "0.1.2",
```

Comment:
Was this downgrade deliberate?

Reply:
Interesting, thanks for catching this! This was not intensional and not sure how I accidentally modified it 😓 will revert this change 👍